autotune_minikube_demo_setup.sh

```diff
@@ -301,6 +306,8 @@ function get_urls() {
 	echo "Info: Autotune Experiments at http://${MINIKUBE_IP}:${AUTOTUNE_PORT}/listExperiments"
 	echo "Info: Autotune Experiments Summary at http://${MINIKUBE_IP}:${AUTOTUNE_PORT}/experimentsSummary"
 	echo "Info: Autotune Trials Status at http://${MINIKUBE_IP}:${AUTOTUNE_PORT}/listTrialStatus"
+	echo "Info: List Layers in autotune http://${MINIKUBE_IP}:${AUTOTUNE_PORT}/query/listStackLayers?deployment_name=autotune&namespace=monitoring"
```

Comment:
Does this link work? Shouldn't we apply autotune yaml to detect autotune itself?

Reply:
I think that link would work when autotune object is applied as per the comment.